### DISCUSSION TITLE
Listing to midi events

### DISCUSSION THREAD

**Discussion by gavriguy** [johnlindquist/kit#759] [Q&A] [answered]
Hi,
First of all thank you for making this project. I found it both useful and mind opening. 

And here is my question: Is there any easy way to trigger script kit by listening t a midi event?
I have a midi controller that I would be happy to use as a poor's man stream deck. I know tools like keyboard maestro support it and I was wondering if it's possible.

**Comment by johnlindquist** [comment, marked as answer]
You can open Kit.app from other apps with:`~/.kit/kar`

For example, if you have a script named `journal`, then pass this into Keyboard Maestro, etc:

```bash
~/.kit/kar journal
```

Here's what it looks like with BetterTouchTool:
![CleanShot 2022-04-19 at 08 07 16](https://user-images.githubusercontent.com/36073/164022887-a5ad55f9-9a35-46ce-b1cf-42fab7598b66.png)

I don't have any current plans to detect devices/events/etc.

**Reply by gavriguy** [reply]
Thank you for your quick response and help
